src/Parser/Hunter/BeastMastery/Modules/Spells/BestialWrath/BestialWrath.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS/index';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import { formatNumber } from 'common/format';
+import getDamageBonus from "Parser/Hunter/Shared/Core/getDamageBonus";
+import StatisticBox from "Main/StatisticBox";
+import SpellIcon from "common/SpellIcon";
+import ITEMS from "common/ITEMS/HUNTER";
+
+const T204P_MODIFIER = 0.15;
+const T20_2P_MODIFIER_PR_STACK = 0.015;
+const BESTIAL_FURY_MODIFIER = 0.15;
+const BESTIAL_WRATH_BASE_MODIFIER = 0.25;
+let BESTIAL_WRATH_TOTAL_MODIFIER = 0;
+
+const AFFECTED_ABILITIES = [
+  SPELLS.KILL_COMMAND.id,
+  SPELLS.COBRA_SHOT.id,
+  SPELLS.MULTISHOT.id,
+];
+
+const debug = false;
+
+class BestialWrath extends Analyzer {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  bonusDmg = 0;
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+    //resets the strength of bestial wrath every time it's cast and adjust for Bestial Fury talent.
+    if (spellId === SPELLS.BESTIAL_WRATH.id) {
+      if (this.combatants.selected.hasTalent(SPELLS.BESTIAL_FURY_TALENT) || this.combatants.selected.hasFinger(ITEMS.SOUL_OF_THE_HUNTMASTER.id)) {
+        BESTIAL_WRATH_TOTAL_MODIFIER = BESTIAL_FURY_MODIFIER + BESTIAL_WRATH_BASE_MODIFIER;
+      } else {
+        BESTIAL_WRATH_TOTAL_MODIFIER = BESTIAL_WRATH_BASE_MODIFIER;
+      }
+    }
+    if (spellId !== SPELLS.KILL_COMMAND.id || spellId !== SPELLS.COBRA_SHOT.id || spellId !== SPELLS.MULTISHOT.id) {
```

Comment:
I assume you defined `AFFECTED_ABILITIES` for this, remember to use it.

Reply:
It breaks if I use it there like I use elsewhere, and I couldn't figure out why 🤔

Reply:
What's "it"? What line were you using, and what error did you get?

Reply:
it being ``AFFECTED_ABILTIES``
line code used elsewhere where it works intentionally: 
``if (AFFECTED_ABILITIES.every(id => spellId !== id))``

Reply:
ok sure but what error do you get when you use it here?